### ISSUE TITLE
[es-dev-server] npm link not handled by the server

### ISSUE THREAD

**Issue by jarrodek** [open-wc/open-wc#664]
I just linked a component from a local folder using `npm link` command. It can see linked folder in proper location however when I run demo page I see 404 error:
```
GET http://localhost:8080/arc-demo-helper/ApiDemoPage.js net::ERR_ABORTED 404 (Not Found)
```
This is the component I am importing in my demo page. The file is there but it is a link. The setup was working perfectly before making the link so it has to be server.

**Comment by LarsDenBakker** [comment]
This is intentional because in a monorepo the same modules are imported with a symlink and with an import to the actual path. If we preserved the symlinks, it would be imported twice.

At the least we should make this behavior configurable, and maybe preserving symlinks should be the default.

**Comment by LarsDenBakker** [comment]
IN https://github.com/open-wc/open-wc/pull/704 you can configure this behavior

**Comment by cristinecula** [comment]
@LarsDenBakker I am having the same issue on v1.12.0:
I have the following directory structure:
* project/
  * app/
  * some-component/

In `app/` I do `npm link some-component` and, when running the server, the imports are now rewritten to `../../../some-component/index.js`, instead of `../../node_modules/some-component/index.js`.

Note that `project` is not a monorepo.

I noticed #704, so I tried it using es-dev-server v1.11.0, but I ran into other issues (some imports were rewritten wrong).
